var/spack/repos/builtin/packages/py-csvkit/package.py

```diff
@@ -35,9 +35,9 @@ class PyCsvkit(PythonPackage):
     version('0.9.1', '48d78920019d18846933ee969502fff6')
 
     depends_on('py-setuptools', type='build')
-    depends_on('py-dateutil', type=('build', 'run'))
-    depends_on('py-dbf', type=('build', 'run'))
+    depends_on('py-dateutil@2.2', type=('build', 'run'))
+    depends_on('py-dbf@0.94.003', type=('build', 'run'))
```

Comment:
I can find places claiming that version >= `0.94.003` is required but not specifically that version. I haven't checked for `py-dateutil` yet but I'm also curious how you noticed version `2.2` is required.

Reply:
In setup.py

``` python
install_requires = [
    'xlrd>=0.7.1',
    'sqlalchemy>=0.6.6',
    'openpyxl==2.2.0-b1',
    'six>=1.6.1',
    'python-dateutil==2.2'
]

if sys.version_info < (2, 7):
    install_requires.append('argparse>=1.2.1')
    install_requires.append('ordereddict>=1.1')
    install_requires.append('simplejson>=3.6.3')

if sys.version_info[0] == 2:
    install_requires.append('dbf==0.94.003')
```

I am not familiar with spack's PythonPackage wrapper, but it doesn't check this at build time.

Reply:
Thanks for the pointer!

Could you do me a favor and modify it to be:

`depends_on('py-dateutil@2.2', type=('build', 'run'), when='@0.9.1')`

?